pkg/config/config.go

```diff
@@ -516,6 +521,11 @@ func InitConfigurationData() error {
 		return err
 	}
 	runningConfig = config
+	var readConfig RunModConfig
+	readConfig, err = ReadRunModConfig(RunModFilePath)
+	if err != nil {
```

Comment:
Change line 526 to
```
if err != nil && !os.IsNotExist(err) {
```
Since we can get there when the file doesn't exist, but that's not really an error (we'll just read from the config file in that case rather than the runmod config file in that case)